chapter3.md

```diff
@@ -64,26 +64,26 @@ Averaging the AICc weights and distinct abundance values across all sites in a d
 
-While the vast majority of the data that we tested was terrestrial (approximately 99%), our data did include 161 fish communities. Of these, the majority were freshwater, rather than marine. However we observed the same general pattern of results for for both freshwater and marine fish communities as we did for the other taxa. This leave open the possibility that some non-biological difference between the two data compilations is driving the differences in the results.
+While the vast majority of the data that we tested was terrestrial (approximately 99%), our data did include 161 fish communities. Of these, the majority were freshwater, rather than marine. However we observed the same general pattern of results for for both freshwater and marine fish communities as we did for the other taxa. This leaves open the possibility that some non-biological difference between the two data compilations is driving the differences in the results.
 
 One potential non-biological explanation for the difference between our results and Connolly et al.'s (2014) results is a difference in the spatial structure of the data: the data from Connolly 2014 is structured in natural spatial groupings, whereas the data that we used in this study is not.  In this study, many of the sites are widely dispersed, or are not regularly dispersed over the landscape.  These differences in spatial grouping may lead to results that are more consistent, due to spatial similarity than our widely dispersed sites.
 
-Another potential non-biological explanation is related to sampling intensity.  It is possible that the way in which marine communities are sampled is different from sampling of terrestrial communities, resulting in differing intensity of sampling that produce different patterns. However, the diversity of data we used covers a broad range of sampling intensities, from a complete censuses (completely sampled trees above some minimum size cutoff; Forest Inventory and Analysis and Gentry), to well sampled but incomplete surveys (e.g., Christmas Bird Count), to less relatively poorly sampled (most of the invertebrate data including spiders and beetles).
+Another potential non-biological explanation is related to sampling intensity.  It is possible that the way in which marine communities are sampled is different from sampling of terrestrial communities, resulting in differing intensity of sampling that produce different patterns. However, the diversity of data we used covers a broad range of sampling intensities, from complete censuses (completely sampled trees above some minimum size cutoff; Forest Inventory and Analysis and Gentry), to well sampled but incomplete surveys (e.g., Christmas Bird Count), to less completely sampled for individuals, but taxonomically less well resolved (e.g. spiders and beetles).
```

Comment:
"to less completely sampled for individuals, but taxonomically less well resolved"
->
"to incompletely taxonomically resolved and incompletely sampled"

I'm not sure exactly what you were trying to communicate with this change. I've tried rewording, but if I've gone in the wrong direction just work on making the language more clear.